src/App.js

```diff
@@ -1,12 +1,52 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
+import axios from 'axios';
 import './App.css';
+import PokemonCard from "./components/PokemonCard";
 
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
+    const [pokemonData, setPokemonData] = useState('');
```

Comment:
Dit zal initieel van het type string zijn als je als default value `''` hanteert. Uiteindelijk wordt het een object dus dan zou ik als default value `undefined` gebruiken en als je het gebruikt eerst checkt dat het niet undefined is